docs/application/vstools/tools/certificate-manager.md

```diff
@@ -36,6 +36,9 @@ To select the certificates used to package your application:
 <a name="creating"></a>
 ## Creating a Certificate Profile
 
+**Note:**
```

Comment:
Note tag format needs to be fixed and "Please" is not a preferred term, suggest rephrasing as: 

>**Note**
>
>It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.